### PR TITLE
Add a new feature to do great things

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Bla bla bla

### CODE REVIEW

File: some_code/Python_file.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 import os
 
 some great code
+
+more great code
```
Comment (smartie2076) [inline some_code/Python_file.py]: I like this code.

**Review by smartie2076 — APPROVED** [review]
Ready to merge

**Comment by catcad** [comment]
i like the great thing

File: some_code/Python_file.py (hunk 1 above)
Comment (Bachibouzouk) [inline some_code/Python_file.py]: explain more what I did here for others

**Comment by JulianBarinton** [comment]
reviewed pull request :)

**Review by catcad — APPROVED** [review]

File: some_code/Python_file.py (hunk 1 above)
Comment (Bachibouzouk) [reply some_code/Python_file.py]:
```suggestion
even more great code
```

**Pull request merged**